pkg/controller/jaeger/account.go

```diff
@@ -0,0 +1,52 @@
+package jaeger
+
+import (
+	"context"
+
+	log "github.com/sirupsen/logrus"
+	"k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/jaegertracing/jaeger-operator/pkg/apis/io/v1alpha1"
+	"github.com/jaegertracing/jaeger-operator/pkg/inventory"
+)
+
+func (r *ReconcileJaeger) applyAccounts(jaeger v1alpha1.Jaeger, desired []v1.ServiceAccount) error {
+	opts := client.MatchingLabels(map[string]string{
+		"app.kubernetes.io/instance":   jaeger.Name,
+		"app.kubernetes.io/managed-by": "jaeger-operator",
+	})
+	list := &v1.ServiceAccountList{}
+	if err := r.client.List(context.Background(), opts, list); err != nil {
+		return err
+	}
+
+	logFields := log.WithFields(log.Fields{
+		"namespace": jaeger.Namespace,
+		"instance":  jaeger.Name,
+	})
+
+	inv := inventory.ForAccounts(list.Items, desired)
+	for _, d := range inv.Create {
+		logFields.WithField("account", d.Name).Debug("creating service account")
```

Comment:
move `logFields.WithField("account", d.Name)` to fields above and `s/creating service account/creating/`

Reply:
Do the same in all new  files in this PR

Reply:
I'll do the documentation, but the logging is a matter of taste... I'll keep it like it is.

Reply:
:+1: good to know